### ISSUE TITLE
questions on the softmax layer in pretraining and also the way of avoiding mismatch during fine tuning

### ISSUE THREAD

**Issue by futaoo** [google-research/bert#199]
1.I just found the weights to be used in softmax layer for predictions is the same as the input embeddings, is it for more effectively training the word embedding? 
2.I also notice that to avoid mismatch between the pretrainning and fine tuning, the mask token is not selected constantly. But if I use the mask as well and keep other tokens the same in fine tuning to get contextual representation for every token, will this be also effective?
